RSSDP/Infrastructure/DisposableManagedObjectBase.cs

```diff
@@ -68,8 +69,7 @@ public string BuildMessage(string header, Dictionary<string, string> values)
         public void Dispose()
         {
             IsDisposed = true;
-
-            Dispose(true);
```

Comment:
Why?

Reply:
I haven't read the warning correctly. Thought I have to replace `Dispose(true)` with `GC.SuppressFinalize(this)` but I think we want both